### PR TITLE
Add missing module docs

### PR DESCRIPTION
**Pull request by sabbyanandan**
- Includes docs for new `source`, `processor`, and `sink` modules
- Corrections to existing module docs

### CODE REVIEW

**Comment by sabbyanandan** [comment]
@artembilan: If you have any other comments or suggestions, please let me know.

File: spring-cloud-stream-modules-docs/src/main/asciidoc/sinks.adoc [hunk 1]
```diff
@@ -1,6 +1,58 @@
...
+
+==== Options
+
+The **$$cassandra$$** $$sink$$ has the following options:
+
+$$compressionType$$:: $$the compression to use for the transport$$ *($$CompressionType$$, default: `NONE`, possible values: `NONE,SNAPPY`)*
+$$consistencyLevel$$:: $$the consistencyLevel option of WriteOptions$$ *($$ConsistencyLevel$$, no default, possible values: `ANY,ONE,TWO,THREE,QUOROM,LOCAL_QUOROM,EACH_QUOROM,ALL,LOCAL_ONE,SERIAL,LOCAL_SERIAL`)*
+$$spring.cassandra.contactPoints$$:: $$the comma-delimited string of the hosts to connect to Cassandra$$ *($$String$$, default: `localhost`)*
+$$entityBasePackages$$:: $$the base packages to scan for entities annotated with Table annotations$$ *($$String[]$$, default: `[]`)*
+$$spring.cassandra.ingestQuery$$:: $$the ingest Cassandra query$$ *($$String$$, no default)*
```
Comment (artembilan) [inline spring-cloud-stream-modules-docs/src/main/asciidoc/sinks.adoc:43]: The `ingestQuery` is a part of `Sink`, so it is without prefix.
See https://github.com/spring-cloud/spring-cloud-stream-modules/blob/master/cassandra-sink/src/main/java/org/springframework/cloud/stream/module/cassandra/CassandraProperties.java for all with prefix.

File: spring-cloud-stream-modules-docs/src/main/asciidoc/sinks.adoc [hunk 2]
```diff
@@ -1,6 +1,58 @@
...
+$$spring.cassandra.initScript$$:: $$the path to file with CQL scripts (delimited by ';') to initialize keyspace schema$$ *($$String$$, no default)*
+$$spring.cassandra.keyspace$$:: $$the keyspace name to connect to$$ *($$String$$, default: `<stream name>`)*
+$$metricsEnabled$$:: $$enable/disable metrics collection for the created cluster$$ *($$boolean$$, default: `true`)*
+$$spring.cassandra.password$$:: $$the password for connection$$ *($$String$$, no default)*
+$$spring.cassandra.port$$:: $$the port to use to connect to the Cassandra host$$ *($$int$$, default: `9042`)*
+$$spring.cassandra.queryType$$:: $$the queryType for Cassandra Sink$$ *($$Type$$, default: `INSERT`, possible values: `INSERT,UPDATE,DELETE,STATEMENT`)*
```
Comment (artembilan) [inline spring-cloud-stream-modules-docs/src/main/asciidoc/sinks.adoc:49]: `queryType` as well.

**Comment by markpollack** [comment]
Resolved by https://github.com/spring-cloud/spring-cloud-stream-modules/commit/8a265f7ffefe186cb24963f518d9f516f5691f11

**Pull request closed without merging**